accumulo/src/test/resources/log4j.properties

```diff
@@ -24,6 +24,6 @@ log4j.appender.stderr.layout=org.apache.log4j.PatternLayout
 log4j.appender.stderr.layout.conversionPattern=%d{yyyy/MM/dd HH:mm:ss} %-5p %c %x - %m%n
 
 # Suppress messages from ZooKeeper
-log4j.logger.com.yahoo.ycsb.db.accumulo=INFO
+log4j.logger.com.yahoo.ycsb.db.accumulo=DEBUG
```

Comment:
Yes. I left the root logger at INFO so there is no user-facing change, but this makes future debugging easier.